.github/workflows/lcov.yml

```diff
@@ -14,10 +14,9 @@ jobs:
       image: vsaglib/vsag:ubuntu
     steps:
       - uses: actions/checkout@v4
-      - name: Set up Python
-        uses: actions/setup-python@v3
-        with:
-          python-version: '3.10'
```

Comment:
there is a pre-install python3.10 in the vsaglib/vsag:ubuntu image. use "actions/setup-python@v3" to install a new will cause another issue: 
```
CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_INCLUDE_DIRS Development.Module)
  (found version "3.10.15")
```